brokerapi/brokers/account_managers/service_account_manager.go

```diff
@@ -202,23 +207,27 @@ type ServiceAccountInfo struct {
 	PrivateKeyData string `json:"PrivateKeyData"`
 }
 
-func ServiceAccountBindInputVariables(serviceName string, defaultWhitelist []string) []broker.BrokerVariable {
-	details := fmt.Sprintf(`The role for the account without the "roles/" prefix.
-		See: https://cloud.google.com/iam/docs/understanding-roles for more details.
-		Note: The default enumeration may be overridden by your operator.`)
-
+// ServiceAccountWhitelistWithDefault holds overridable whitelists with default values.
```

Comment:
The name in this comment block doesn't seem to match up with the function itself.